compiler/quilt/tools/command.py

```diff
@@ -1447,3 +1448,204 @@ def load(pkginfo):
     """functional interface to "from quilt.data.USER import PKG"""
     # TODO: support hashes/versions/etc.
     return _load(pkginfo)[0]
+
+def export(package, output_path='.', filter=lambda x: True, mapper=lambda x: x, force=False):
```

Comment:
Is the mapper needed for tensor flow? If not, let's remove it to keep the API as simple as possible.